packages/api/src/controllers/user.ts

```diff
@@ -60,6 +62,39 @@ async function findUserByEmail(email: string, useReplica = true) {
   return users[0];
 }
 
+const frontendUrl = (
+  {
+    headers: { "x-forwarded-proto": proto },
+    config: { frontendDomain },
+  }: Request,
+  path: string
+) => `${proto || "http"}://${frontendDomain}${path}`;
+
+const unsubscribeUrl = (req: Request) => frontendUrl(req, "/contact");
```

Comment:
This is also in the other sendGrid notification pr we merged, we should at some point put a correct unsubscribe url for `{{{unsubscribe}}}` in the sendGrid template? Or at least not use an "unsubscribe" link into the email template but a "contact us" one, as it's a notification for the service and not a newsletter